components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/util/OAuth2Util.java

```diff
@@ -2758,6 +2739,9 @@ public static boolean validateIdToken(String idToken) {
                 log.debug("Error occurred while validating id token signature.");
             }
             return false;
+        }  catch (IdentityKeyStoreResolverException e) {
+            log.error("Error occurred while validating id token signature.");
```

Comment:
Log exception

Reply:
Part of legacy code

Reply:
https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/2524/commits/ebfefe5fed433ef382ac5ff5642e9710fb046c3e